### DISCUSSION TITLE
Can the columns of long listing (-l) be made fixed-width even when STDOUT is not a tty?

### DISCUSSION THREAD

**Discussion by clharrison** [eza-community/eza#858] [Q&A, Howto, Help]
Sorry if I've overlooked anything prior to posting. I'm writing a Perl tool that fires off an **eza -l** command via **qx()** and reads the output into a variable - therefore, eza knows its STDOUT is not to a tty. I use **--color=always** and **--classify=always** to force those features, but I lose the fixed-width columns I would get if running **eza -l** from the terminal. I assume that's because STDOUT is not a tty, but there seems to be no way to override fixed columns the way I can force coloring and classification.

Have I missed an option?

**Update**
I noticed that using eza's **--header** option forces fixed-width columns with piped output, except that I don't want headers to be printed. Perhaps #766 will give me what I want?

**Comment by Omnikron13** [comment]
I believe `eza --header --color=always --classify=always | tail -n +2` would work around the issue, at least.
